app/views/admin/inbound_api_configurations/_form.haml

```diff
@@ -0,0 +1,11 @@
+.row
+  .col-sm-8
+    - url = admin_inbound_api_configurations_path
+    = simple_form_for [:admin, @conf], url: url do |f|
+      = f.error_notification
+      .form-inputs
+        = f.input :external_system_name
+        = f.input :internal_system_name
```

Comment:
This will be a select_two input with a collection, right? I think for now the collection just contains `referral` and `involvement` which need to be stored as constants somewhere

Reply:
I think you'll need `as: :select_two` if we want select twos, which we probably do eventually, but totally doesn't matter for now.

Reply:
I didn't know how these were to be wired up, so I didn't implement anything. Would these be a constant on this new model or some other pre-existing model? Perhaps we could use an `InternalSystem` model/table. Then we could enable/disable them. I don't have a preference, since I don't know the landscape well. Let me know what you think, @gigxz

Reply:
Yeah, I hadn't really thought that through yet. I agree that they should be stored in a table so that they can be enabled/disabled per installation. I think it can be a simple table with a name and an active boolean. Then this dropdown would only show systems that are active.

We should probably seed the new table on deploy to contain all the apis we have (with active: false). As far as the constants for the system names themselves (`"referral", "involvement"`) would it make sense to just store them as constants on each controller respectively - like `HmisExternalApis::ReferralsController::INTERNAL_SYSTEM_NAME`? That controller is where that value will be _used_, after all, since the controller will be looking up the incoming API keys against that `internal_system_name`.

How does that sound @yakloinsteak @eanders?

I feel so-so about InternalSystem as a table/model name.. but my other idea was `HmisExternalApis::PublicApi`/`hmis_public_apis` which might be more confusing since they are referred to as "internal" elsewhere.